stress/shell/src/main/java/alluxio/stress/cli/MaxFileBench.java

```diff
@@ -0,0 +1,222 @@
+    mainInternal(args, new MaxFileBench());
+  }
+
+  @Override
+  public String getBenchDescription() {
+    List<String> descLines = new ArrayList<>(Arrays.asList(
+        "MaxFile. Creates files until no more files can be created.",
+        "This stressbench ignore the following options and sets its own values as follows:"
+    ));
+    for (int i = 0; i < mDefaultParams.size(); i += 2) {
+      descLines.add(String.format("%s=%s", mDefaultParams.get(i), mDefaultParams.get(i + 1)));
+    }
+    return Strings.join("\n\t", descLines);
+  }
+
+  @Override
+  protected void parseParameters(String[] args) {
+    List<String> argsList = new ArrayList<>(Arrays.asList(args));
+    argsList.addAll(mDefaultParams);
+
+    JCommander jc = new JCommander(this);
+    jc.setAllowParameterOverwriting(true);
+    jc.setProgramName(this.getClass().getSimpleName());
+    try {
+      jc.parse(argsList.toArray(new String[0]));
+      if (mBaseParameters.mHelp) {
+        System.out.println(getBenchDescription());
+        jc.usage();
+        System.exit(0);
+      }
+    } catch (Exception e) {
+      LOG.error("Failed to parse command: ", e);
+      System.out.println(getBenchDescription());
+      jc.usage();
+      throw e;
+    }
+  }
+
+  @Override
+  public MasterBenchTaskResult runLocal() throws Exception {
+    ExecutorService service = ExecutorServiceFactories.fixedThreadPool("maxfile-bench-thread",
+        mParameters.mThreads).create();
+
+    List<Callable<Void>> callables = new ArrayList<>(mParameters.mThreads);
+    for (int i = 0; i < mParameters.mThreads; i++) {
+      callables.add(new AlluxioNativeMaxFileThread(i, mCachedNativeFs[i % mCachedNativeFs.length]));
+    }
+    LOG.info("Starting {} bench threads", callables.size());
+    long startMs = CommonUtils.getCurrentMs();
+    service.invokeAll(callables, FormatUtils.parseTimeSize(mBaseParameters.mBenchTimeout),
+        TimeUnit.MILLISECONDS);
+    mTotalResults.setDurationMs(CommonUtils.getCurrentMs() - startMs);
+    LOG.info("Bench threads finished");
+
+    service.shutdownNow();
+    service.awaitTermination(30, TimeUnit.SECONDS);
+
+    return mTotalResults;
+  }
+
+  private final class AlluxioNativeMaxFileThread implements Callable<Void> {
+    private static final int INIT_WAIT_TIME_MS = 100;
+    private static final int MAX_WAIT_TIME_MS = 15_000;
+    private static final int OPERATION_TIMEOUT_MS = 2_000;
+
+    private int mWaitTimeMs = INIT_WAIT_TIME_MS;
+    private final ExecutorService mExecutor = Executors.newSingleThreadExecutor();
+    private final MasterBenchTaskResult mResult = new MasterBenchTaskResult();
+
+    private final int mId;
+    private final FileSystem mFs;
+    private final Path mBasePath;
+    private final Path mFixedBasePath;
+
+    AlluxioNativeMaxFileThread(int id, FileSystem fs) {
+      mId = id;
+      mFs = fs;
+      mBasePath =
+          new Path(PathUtils.concatPath(mParameters.mBasePath, "files", mId));
```

Comment:
Let's consolidate the constant with StressMasterBench, like "files" and "fixed", instead of put String directly here